src/standalone/Vegetation/solar_induced_fluorescence.jl

```diff
@@ -68,14 +68,78 @@ function update_SIF!(p, Y, sif_model::Lee2015SIFModel, canopy)
     planck_h = LP.planck_constant(earth_param_set)
     N_a = LP.avogadro_constant(earth_param_set)
     (; λ_γ_PAR,) = canopy.radiative_transfer.parameters
-    energy_per_mole_photon_par = planck_h * c / λ_γ_PAR * N_a
     T_freeze = LP.T_freeze(earth_param_set)
     R = LP.gas_constant(earth_param_set)
-
-    (; ΔHJmax, To, θj, ϕ) = canopy.photosynthesis.parameters
     sif_parameters = sif_model.parameters
+
+    APAR = @. lazy(compute_APAR(f_abs_par, par_d, λ_γ_PAR, c, planck_h, N_a))
+    _update_SIF_dispatch!(
```

Comment:
Im not sure about this _ notation. What does it indicate? If we add it here, do we need to add it to all other update! functions?

Reply:
A leading `_` is usually reserved for internal (not user-facing functions) in Julia. This function is technically internal, but as Kat points out we have many internal `update!` functions that don't use this syntax - I think it's better to be consistent with those